wallet/wallet.c

```diff
@@ -31,6 +31,31 @@
 /* 12 hours is usually enough reservation time */
 #define RESERVATION_INC (6 * 12)
 
+/* Possible channel state */
+enum channel_state_db {
```

Comment:
Nit: the `_db` suffix suggested to me that this is something we store in the DB, which requires extra care since those values are persistent. However it's just an internal value that doesn't reach the DB, so I suggest dropping the `_db` suffix.

Reply:
Maybe a more descriptive name would be `channel_stat_bucket`?

Reply:
Concept Ack.

`channel_stat_bucket` much better